### PR TITLE
Issue #555 ; use theme function instead of additional user-picture.tpl.p...

### PR DESCRIPTION
**Pull request by serundeputy**
This solves: https://github.com/backdrop/backdrop-issues/issues/555

### CODE REVIEW

File: core/modules/user/templates/user-profile.tpl.php [hunk 1]
```diff
@@ -32,4 +32,6 @@
 ?>
 <article class="profile"<?php print backdrop_attributes($attributes); ?>>
   <?php print render($user_profile); ?>
+  <?php print render($user_picture); ?>
 </article>
+
```
Comment (quicksketch) [inline core/modules/user/templates/user-profile.tpl.php:37]: Hm, looks like $user_picture is getting printed twice in here now?

File: core/modules/user/user.module [hunk 2]
```diff
@@ -59,7 +59,7 @@ function user_theme() {
   return array(
     'user_picture' => array(
       'variables' => array('account' => NULL),
-      'template' => 'templates/user-picture',
+      'template' => 'templates/user-profile',
```
Comment (quicksketch) [inline core/modules/user/user.module:62]: This line just needs to be removed if we're deleting the template.

File: core/modules/user/templates/user-profile.tpl.php (hunk 1 above)
Comment (serundeputy) [inline core/modules/user/templates/user-profile.tpl.php:37]: Isn't one $user_profile and one $user_picture?  I think it is correct no?

File: core/modules/user/user.module (hunk 2 above)
Comment (serundeputy) [inline core/modules/user/user.module:62]: At first I took out this line, but something went wrong so I pointed at the user-profile tpl

File: core/modules/user/templates/user-profile.tpl.php (hunk 1 above)
Comment (quicksketch) [inline core/modules/user/templates/user-profile.tpl.php:37]: Doh, yes sorry. I'll need to test this locally.

**Comment by serundeputy** [comment]
@quicksketch: I took a look at these tonight; false positives?  Looks like the markup the tests are looking for is present in each case.

**Comment by quicksketch** [comment]
I think the test failures are legit. I'd try enabling the Testing module on your local and then running the individual tests that are failing under Configuration > Development > Testing. Then look at the verbose log messages in the result, that will show you the HTML that is being tested.

**Pull request closed without merging**